app/lib/funding/parsers/base.rb

```diff
@@ -12,8 +12,8 @@ def expected_headers
           raise(NotImplementedError("implement in subclass"))
         end
 
-        def to_attributes(file_path:)
-          csv = CSV.open(file_path, headers: true)
+        def to_attributes(funding_upload)
+          csv = CSV.parse(funding_upload.csv_data, headers: true)
```

Comment:
I think we still need to support a regular csv file path for the rake task incase it needs to be done manually. We could check for either and work off that?